src/modules/LanItems.rb

```diff
@@ -1193,7 +1186,8 @@ def Read
     # 3) variables which keeps (some of) attributes of the current item (= item
     # which is being pointed by the iterator)
     def reset_cache
-      LanItems.Items = {}
+      # FIXME: what is suitable replacement with new config storage?
```

Comment:
probably something like reassign `:yast` config again with system one? but here it makes config basically completely empty. What is usecase?

Reply:
leftover from the old code when it handled cases when some variables were left initialized to some value and not properly refreshed. Hopefully will not be needed in the bright future.